src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -466,6 +471,7 @@ def load_arguments(self, _):
         c.argument('nat_gateway_managed_outbound_ip_count', type=int, validator=validate_nat_gateway_managed_outbound_ip_count)
         c.argument('nat_gateway_idle_timeout', type=int, validator=validate_nat_gateway_idle_timeout)
         c.argument('network_dataplane', arg_type=get_enum_type(network_dataplanes))
+        c.argument('network_policy', arg_type=get_enum_type(network_policies))
```

Comment:
adding enum validation is nice, but could we defer that to a later PR? Want to avoid referencing "none" until it's ready for people to try it.

Reply:
Removed enum validation